src/main/java/com/sonyericsson/jenkins/plugins/bfa/statistics/StatisticsLogger.java

```diff
@@ -126,7 +126,7 @@ public void run() {
             Node node = build.getBuiltOn();
             String nodeName = node.getNodeName();
             int timeZoneOffset = TimeZone.getDefault().getRawOffset();
-            String master = "";
+            String master;
```

Comment:
OK
I normally always intialize with the empty string in case of future expansions (I've had too many `nullSometext` printouts in the past). But if it makes you feel better we can have it like this as well ;)